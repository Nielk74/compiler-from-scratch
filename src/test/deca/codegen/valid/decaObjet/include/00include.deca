// Description:
//   test sur 2 inclusion de fichier
//
// Resultats:
//    00include.expected
//
// Historique:
//    cree le 13/06/2022

#include "include.decah"
#include "include1.decah"
    println("x + 1 in main = " , x+ 1);
    println("hello world!");
}