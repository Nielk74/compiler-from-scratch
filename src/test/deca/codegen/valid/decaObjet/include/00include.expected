x in imported file: 1
x in imported file test.deca: 1
x + 1 in main = 2
hello world!
